### PR TITLE
getters arp,ntp,ping

### PR DESCRIPTION
**Pull request by ppieprzycki**
Hello first part of getters to this new driver.

Sure we will have to complete unit tests before driver release 

Simple outputs from functions

#get_arp_table()
[{u'interface': u'eth0', u'ip': u'10.0.2.6', u'mac': u'00:00:00:00:00:00', u'age': 0.0}, {u'interface': u'eth0', u'ip': u'10.0.2.3', u'mac': u'52:54:00:12:35:03', u'age': 0.0}, {u'interface': u'eth0', u'ip': u'10.0.2.2', u'mac': u'52:54:00:12:35:02', u'age': 0.0}]

#get_ntp_stats()
[{u'jitter': 0.0, u'synchronized': False, u'offset': 0.0, u'referenceid': u'.INIT.', u'remote': u'193.25.222.240', u'reachability': 0, u'when': u'0', u'delay': 0.0, u'hostpoll': 1024, u'stratum': 16, u'type': u'u'}, {u'jitter': 0.0, u'synchronized': False, u'offset': 0.0, u'referenceid': u'.INIT.', u'remote': u'188.112.42.5', u'reachability': 0, u'when': u'0', u'delay': 0.0, u'hostpoll': 1024, u'stratum': 16, u'type': u'u'}, {u'jitter': 0.0, u'synchronized': False, u'offset': 0.0, u'referenceid': u'.INIT.', u'remote': u'193.70.94.182', u'reachability': 0, u'when': u'0', u'delay': 0.0, u'hostpoll': 1024, u'stratum': 16, u'type': u'u'}, {u'jitter': 0.0, u'synchronized': False, u'offset': 0.0, u'referenceid': u'.INIT.', u'remote': u'188.165.17.91', u'reachability': 0, u'when': u'0', u'delay': 0.0, u'hostpoll': 1024, u'stratum': 16, u'type': u'u'}]

#get_ntp_peers()
{u'193.25.222.240': {}, u'193.70.94.182': {}, u'188.165.17.91': {}, u'188.112.42.5': {}}

#ping(destination="8.8.8.8", size="56", count=4) successful ping

{u'success': {u'packet_loss': 0, u'rtt_stddev': 2.504, u'rtt_min': 36.729, u'results': [{u'rtt': u'42.9', u'ip_address': u'8.8.8.8'}, {u'rtt': u'37.7', u'ip_address': u'8.8.8.8'}, {u'rtt': u'37.3', u'ip_address': u'8.8.8.8'}, {u'rtt': u'36.7', u'ip_address': u'8.8.8.8'}], u'rtt_avg': 38.715, u'rtt_max': 42.997, u'probes_sent': 4}}

#ping(destination="8.8.8.80", size="50") failed ping 
{u'success': {u'packet_loss': 10, u'rtt_stddev': None, u'rtt_min': None, u'results': [], u'rtt_avg': None, u'rtt_max': None, u'probes_sent': 10}}

### CODE REVIEW

**Comment by GGabriele** [comment]
@ppieprzycki great job here! Can you please rebase to solve the conflic issue?

File: napalm_cumulus/cumulus.py [hunk 1]
```diff
@@ -191,3 +191,188 @@ def get_facts(self):
...
+                    "hostpoll": int(hostpoll),
+                    "reachability": int(reachability),
+                    "delay": float(delay),
+                    "offset": float(offset),
+                    "jitter": float(jitter)
+                })
+
+        return ntp_stats
+
+    def get_ntp_peers(self):
```
Comment (mirceaulinic) [inline napalm_cumulus/cumulus.py]: Does cumulus know to differentiate between NTP servers and NTP peers?

Comment (ppieprzycki) [inline napalm_cumulus/cumulus.py]: Thanks for suggestion. Seems that this part needs to be improved. 
Command ntpq -np actually lists servers & peers. I cannot find param for listing only peers  


So if we want only peers we need look into text config file.  

I'll look further into that 

ntpq> peers
     remote           refid      st t when poll reach   delay   offset  jitter
==============================================================================
 194.29.130.252  .INIT.          16 u    -   64    0    0.000    0.000   0.000
 159.253.242.123 .INIT.          16 u    -   64    0    0.000    0.000   0.000
 213.199.225.40  .INIT.          16 u    -   64    0    0.000    0.000   0.000
 193.70.94.182   .INIT.          16 u    -   64    0    0.000    0.000   0.000
 10.100.100.162  .INIT.          16 u    -   64    0    0.000    0.000   0.000
 10.100.100.160  194.146.251.100  2 u   34   64    1    1.543  1104837   0.031

vagrant@cumulus:~$ sudo cat /etc/ntp.conf | egrep 'peer|server'
[...]
server 0.cumulusnetworks.pool.ntp.org iburst
server 1.cumulusnetworks.pool.ntp.org iburst
server 2.cumulusnetworks.pool.ntp.org iburst
peer 3.cumulusnetworks.pool.ntp.org iburst
peer 10.100.100.162
server 10.100.100.160

**Comment by ppieprzycki** [comment]
I've removed get_ntp_peers def. Command ntpq don't provide info about who is server and who has peer capability.

**Pull request merged**